docs/sources/plugins/marketplace.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Plugin marketplace"
+aliases = ["/docs/grafana/latest/plugins/marketplace/"]
+weight = 1
++++
+
+Marketplace for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Access is limited to Grafana Admins.
```

Comment:
> Access is limited to Grafana Admins

Access to what? The entire app or installing/uninstalling? Can Viewers at least browse plugins?